upi/openstack/security-groups.yaml

```diff
@@ -93,6 +93,22 @@
       port_range_min: 6081
       port_range_max: 6081
 
+  - name: 'Create master-sg rule "IPsec IKE"'
+    os_security_group_rule:
+      security_group: "{{ os_sg_master }}"
+      protocol: udp
+      remote_ip_prefix: "{{ os_subnet_range }}"
+      port_range_min: 500
+      port_range_max: 500
+
+  - name: 'Create master-sg rule "IPsec NAT-T"'
+    os_security_group_rule:
+      security_group: "{{ os_sg_master }}"
+      protocol: udp
```

Comment:
Just to note that, here, I was unable to add "esp" as a protocol. That looks like an issue with https://docs.ansible.com/ansible/2.10/collections/openstack/cloud/security_group_rule_module.html. I raised a Bugzilla against this: https://bugzilla.redhat.com/show_bug.cgi?id=1916474

Reply:
However, if you know a way to do this, please let me know.

Reply:
I'm afraid that I do not. I don't know too much about OpenStack.